admin/src/components/ContributionMessages/slots/ContributionMessagesListItem.vue

```diff
@@ -25,3 +29,19 @@ export default {
   },
```

Comment:
if `message` is required it don't needs a default, which has to be tested.

PS: I know that this is not your code, but we should change it I guess …

Reply:
Good point.